src/components/RoundSelector.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box } from "@chakra-ui/react";
+import { trpc } from "utils/trpc";
+import { useGlobalContext } from "contexts/GlobalContext";
+
+export function RoundSelector({ handleChange = () => {} }: any): JSX.Element {
+  const { requestedRound, requestRound } = useGlobalContext();
+
+  const roundList = trpc.rounds.roundlist.useQuery(undefined, {
+    refetchOnWindowFocus: false,
+  }).data?.data ?? {
+    rounds: [],
+    latest: 12,
```

Comment:
why 12?

Reply:
no special reason - changed for testing only